README.md

```diff
@@ -883,6 +888,83 @@ Allows matching based on Time-To-Live (TTL) value. Valid values are type Integer
 | `ttl => '128'`
 
 --
+### Type: cisco_bfd_global
+
+Manages configuration of a BFD (Bidirectional Forwarding Detection) instance.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N3k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.4.0                  |

```

Comment:
I did not put N8k because it is not there in the README.md file at all. I thought we can add this globally sometime in the future. I mentioned it in the PR comments. If you want me to add this just to bfd_global, I can do it but I won't be able to add it to the support matrix.